api/CSSFontFeatureValuesRule.json

```diff
@@ -31,7 +31,10 @@
           "experimental": false,
           "standard_track": true,
           "deprecated": false
-        }
+        },
+        "tags": [
+          "web-features:font-variant-alternates"
```

Comment:
This and the next file is from https://github.com/web-platform-dx/web-features/pull/772